src/main/java/org/broadinstitute/consent/http/db/mapper/DarCollectionSummaryReducer.java

```diff
@@ -44,6 +45,11 @@ public void accumulate(Map<Integer, DarCollectionSummary> map, RowView rowView)
           summary.addReferenceId(darReferenceId);
         }
 
+        darParentId = rowView.getColumn("dar_parent_id", Integer.class);
+        if (Objects.nonNull(darParentId)){
+          summary.addParentToReferenceId(darParentId, darReferenceId);
+        }
```

Comment:
`RowMapperHelper` has `hasOptionalColumn()` which avoid the need for a local variable and removes the need to specify the field name in two places.
```suggestion
        hasOptionalColumn(rowView, "dar_parent_id", Integer.class)
            .ifPresent(darParentId -> summary.addParentToReferenceId(darParentId, darReferenceId));
```